core/src/main/java/org/infinispan/container/entries/VersionedRepeatableReadEntry.java

```diff
@@ -21,12 +21,12 @@
  * @author Manik Surtani
  * @since 5.1
  */
-public class ClusteredRepeatableReadEntry extends RepeatableReadEntry implements Versioned {
+public class VersionedRepeatableReadEntry extends RepeatableReadEntry implements Versioned {
```

Comment:
Not in this PR; I could remove it along with deprecating optimistic txs w/o wsc.